### PR TITLE
webgui: adjust v7 classes drawing, synchronize jsroot code

### PR DESCRIPTION
**Pull request by linev**
Current v7 canvas cannot be stored in the file - I/O for shared_ptr<> not working

Because of that remove transfer of Pad/Canvas to the clients.
Actually, it was not required at all.
Also adjust extraction of attributes (line color) from the drawables.

Repair all relevant macros in tutorials/v7

Update JSROOT, which now supports LZ4

### CODE REVIEW

File: tutorials/v7/draw.cxx [hunk 1]
```diff
@@ -46,7 +46,7 @@ void draw()
 
    // Create a canvas to be displayed.
    auto canvas = Experimental::TCanvas::Create("Canvas Title");
-   canvas->Draw(pHist).SetLineColor(Experimental::TColor::kRed);
+   canvas->Draw(pHist)->GetOptions().SetLineColor(Experimental::TColor::kRed);
```
Comment (Axel-Naumann) [inline tutorials/v7/draw.cxx]: `GetOptions()` shouldn't be needed anymore since yesterday's changes.

File: tutorials/v7/draw_th1.cxx [hunk 2]
```diff
@@ -41,8 +41,8 @@ void draw_th1() {
 
   // Create a canvas to be displayed.
   auto canvas = Experimental::TCanvas::Create("Canvas Title");
-  canvas->Draw(pHist).SetLineColor(Experimental::TColor::kRed);
-  canvas->Draw(pHist2).SetLineColor(Experimental::TColor::kBlue);
+  canvas->Draw(pHist)->GetOptions().SetLineColor(Experimental::TColor::kRed);
+  canvas->Draw(pHist2)->GetOptions().SetLineColor(Experimental::TColor::kBlue);
```
Comment (Axel-Naumann) [inline tutorials/v7/draw_th1.cxx]: See above

File: tutorials/v7/fitpanel.cxx [hunk 3]
```diff
@@ -68,7 +68,7 @@ void fitpanel() {
    pHist->Fill(3);
 
    auto canvas = Experimental::TCanvas::Create("Canvas Title");
-   canvas->Draw(pHist).SetLineColor(Experimental::TColor::kRed);
+   canvas->Draw(pHist)->GetOptions().SetLineColor(Experimental::TColor::kRed);
```
Comment (Axel-Naumann) [inline tutorials/v7/fitpanel.cxx]: See above.

File: tutorials/v7/text.cxx [hunk 4]
```diff
@@ -32,7 +32,7 @@ void text()
    auto canvas = Experimental::TCanvas::Create("Canvas Title");
 
   auto text = std::make_shared<ROOT::Experimental::TText>("Hello World");
-  canvas->Draw(text).SetFillColor(Experimental::TColor::kRed);
+  canvas->Draw(text)->GetOptions().SetFillColor(Experimental::TColor::kRed);
```
Comment (Axel-Naumann) [inline tutorials/v7/text.cxx]: Ditto.

**Comment by linev** [comment]
> This shouldn't compile with the current master? Draw() doesn't need GetOptions() anymore.

I took version from last Friday.
I closing this PR.
New one will follow once I test it with newest code

**Pull request closed without merging**